test/verify/check-image

```diff
@@ -192,6 +192,84 @@ class TestImage(composerlib.ComposerCase):
         # collect code coverage result
         self.check_coverage()
 
+    def testOSTree(self):
+        b = self.browser
+        m = self.machine
+        if (m.image == "fedora-32"):
+            image_type_ostree = "fedora-iot-commit"
+        elif (m.image == "rhel-83"):
+            image_type_ostree = "rhel-edge-commit"
+        else:
+            return
```

Comment:
@henrywang I would appreciate your advice on this test. We should only build an ostree image on compatible systems. Do you have any recommendations?